editoast/src/main.rs

```diff
@@ -86,6 +86,18 @@ async fn runserver(
     // Setup shared states
     let infra_caches = Data::new(CHashMap::<i64, InfraCache>::default());
 
+    // Setup sentry
+    if args.sentry_dsn.is_some() && args.sentry_env.is_some() {
+        let _guard = sentry::init((
```

Comment:
I believe the guard should only be dropped when the scope of main dies. Please wrap the guard in an Option and let it live as long as the program. You could also move this to a subroutine

Reply:
looks like it is working this way with some unsafe code in the crate :confused: